README.md

```diff
@@ -2,18 +2,6 @@
 
 Unofficial native OpenRepos.net client for Sailfish OS
 
-[![Release](https://img.shields.io/github/release/mentaljam/harbour-storeman.svg)]()
```

Comment:
This shield updates corresponding to the latest release on GitHub. But I often forget to mark tags as releases. Nevertheless this badge should be left on its place.

Reply:
> But I often forget to mark tags as releases. 

That's why I suggested to reduce the manual steps (of a release) as much as possible.  As you already missed to tag a couple of releases correctly, this step is obviously not part of your release process.  Which is O.K. IMO, but then you should not rely on this information here.

> Nevertheless this badge should be left on its place.

Providing incorrect information is always worse than omitting it (especially when the correct information is easily available), IMO.

My aim was to **streamline** the README.md (by showing only static information) and to make the release process easier for you, but not to introduce another mandatory step in your release process.